ui/documents/templates/semantic-ui/documents/Main.jinja

```diff
@@ -1,5 +1,17 @@
 {#def metadata, ui, record, extra_context #}
-<h1 class="ui header">{{ metadata.title if metadata.title else _("Title not provided") }}</h1>
+{% set ns = namespace(localized_title=None)%}
```

Comment:
`metadata.title` value gets "lost" here, replaced with alternative, never to be seen anywhere, is there a request to try to localize main record title? Shouldn't the main original title always be visible somewhere?

Reply:
@mirekys It is OK, that was my interpretation of the request, though I am not 100% sure, if I understood correctly:)